hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/metadata/SCMDBDefinition.java

```diff
@@ -93,6 +104,6 @@ public String getLocationConfigKey() {
   @Override
   public DBColumnFamilyDefinition[] getColumnFamilies() {
     return new DBColumnFamilyDefinition[] {DELETED_BLOCKS, VALID_CERTS,
-        REVOKED_CERTS, PIPELINES, CONTAINERS};
```

Comment:
Since we add a new table in SCM DB,  we need a JIRA to track how to handle back compatibility during upgrade non HA SCM to HA SCM.

Reply:
https://issues.apache.org/jira/browse/HDDS-4635